db/migrate/20181004013839_add_composite_index_to_news_coin_mention.rb

```diff
@@ -0,0 +1,22 @@
+class AddCompositeIndexToNewsCoinMention < ActiveRecord::Migration[5.1]
+  disable_ddl_transaction!
+
+  def change
+    reversible do |dir|
+      dir.up do
+        # remove duplicates
+        execute <<-SQL
```

Comment:
Actually we need to wrap this in a safety_assured { ... } block due to the Strong Migrations gem:
https://github.com/ankane/strong_migrations